ml_peg/app/utils/register_callbacks.py

```diff
@@ -138,6 +141,84 @@ def update_scores_store(
         return scores_data
 
 
+def register_benchmark_to_category_callback(
```

Comment:
I need to look at this more carefully, but my gut question is to check why this needs to be different to the existing callback we have?

In either case, we have a list of tables (from categories, or from benchmarks in a category), and these are used to populate a summary table.

Is anything different because they're all in the same tab?